spec/controllers/api/v1/events_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Api::V1::EventsController, type: :controller do
+
+  it "index" do
+    VCR.use_cassette("events") do 
+       Event.create(name: "Book Drive for kids and Book Club for adults!",
+                 short_description: "Help us harvest the community garden. Then taste the freshest produce ever!",
```

Comment:
Line is too long. [114/80]<br>Align the elements of a hash literal if they span more than one line.